src/voterData/util/voterOutreach.util.ts

```diff
@@ -0,0 +1,444 @@
+              name: 'scroll',
+            },
+            {
+              type: 'text',
+              text: ' AI-Generated Script:',
+              style: {
+                bold: true,
+              },
+            },
+          ],
+        },
+        {
+          type: 'rich_text_preformatted',
+          elements: [
+            {
+              type: 'text',
+              text: String(messagingScript),
+            },
+          ],
+        },
+      ],
+    },
+    {
+      type: 'divider',
+    },
+    {
+      type: 'rich_text',
+      elements: [
+        {
+          type: 'rich_text_section',
+          elements: [
+            {
+              type: 'emoji',
+              name: 'speech_balloon',
+            },
+            {
+              type: 'text',
+              text: ' Message From User:\n',
+              style: {
+                bold: true,
+              },
+            },
+          ],
+        },
+        {
+          type: 'rich_text_quote',
+          elements: [
+            {
+              type: 'text',
+              text: String(message),
+            },
+          ],
+        },
+      ],
+    },
+    {
+      type: 'rich_text',
+      elements: [
+        {
+          type: 'rich_text_section',
+          elements: [
+            {
+              type: 'emoji',
+              name: 'busts_in_silhouette',
+            },
+            {
+              type: 'text',
+              text: ' Audience Selection:',
+              style: {
+                bold: true,
+              },
+            },
+          ],
+        },
+      ],
+    },
+    {
+      type: 'rich_text',
+      elements: [
+        {
+          type: 'rich_text_list',
+          style: 'bullet',
+          elements: [
+            ...formattedAudience,
+            {
+              type: 'rich_text_section',
+              elements: [
+                {
+                  type: 'text',
+                  text: 'Audience Request: ',
+                  style: {
+                    bold: true,
+                  },
+                },
+                {
+                  type: 'text',
+                  text: audienceRequest ? String(audienceRequest) : 'N/A',
+                },
+              ],
+            },
+          ],
+        },
+      ],
+    },
+    {
+      type: 'divider',
+    },
+    imageUrl
+      ? {
+          type: 'rich_text',
+          elements: [
+            {
+              type: 'rich_text_section',
+              elements: [
+                {
+                  type: 'emoji',
+                  name: 'floppy_disk',
+                },
+                {
+                  type: 'text',
+                  text: ' Image File:',
+                  style: {
+                    bold: true,
+                  },
+                },
+              ],
+            },
+          ],
+        }
+      : undefined,
+    imageUrl
+      ? {
+          type: 'section',
+          text: {
+            type: 'mrkdwn',
```

Comment:
Please change the `type` string constants here to reference the `SlackMessageType` enum here: https://github.com/thegoodparty/gp-api/blob/543f5197e810b3a8c1a37b5c7cd7a5ebdc802ab5/src/shared/services/slackService.types.ts#L11-L22